benchmarks/Carter.Benchmarks/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Carter.Benchmarks
+{
+    using BenchmarkDotNet.Running;
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args);
```

Comment:
Hi @ritasker 

I saw your question at https://gitter.im/dotnet/BenchmarkDotNet

If you are getting timeout on a CI then one thing you could consider would be to change default BenchmarkDotNet settings.

BDN does not assume anything about the code (herutistics/caching strategies) and it runs at least 6 warmup iterations. You could set it to 1.
Moreover, the default IterationTime is 500ms, you could set it to 250ms.
Another important thing is that BDN runs the benchmark until the results become stable or it reaches the default max iteration count (100). You could set it to 20.

```cs
BenchmarkSwitcher.FromAssembly(typeof(Program).Assembly).Run(args, CreateConfig());

private static IConfig CreateConfig()
{
    var job = Job.Default
        .WithWarmupCount(1)
        .WithIterationTime(TimeInterval.FromMilliseconds(250))
        .WithMaxIterationCount(20);

    return DefaultConfig.Instance
        .With(job.AsDefault()); // tell BDN that this are the default settings (so things passed as console line arguments are applied to this job)
}
```